### PR TITLE
Use actual `.py` for example sources

### PR DESCRIPTION
**Pull request by naveen521kk**
This is a lot better than having everything in a single
`.json` file. The comments in first few lines contain
the necessary metadata required while rendering and
running `scripts/render-manim-examples.py` will create
the previously exists `manim_examples.json` automatically.

Fixes https://github.com/ManimCommunity/manim-website/issues/37

### CODE REVIEW

File: examples/ContinuousMotion.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# output: ContinuousMotion
+# name: ContinuousMotion
+# visible: False
+# type: video
+
```
Comment (Darylgolden) [inline examples/ContinuousMotion.py:5]: Does not have an import statement.

File: examples/SquareToCircle.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# output: SquareToCircle
+# name: SquareToCircle
+# visible: False
+# type: video
+
```
Comment (Darylgolden) [inline examples/SquareToCircle.py:5]: Does not have an import statement.

File: examples/WarpSquare.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# output: WarpSquare
+# name: WarpSquare
+# visible: False
+# type: video
+
```
Comment (Darylgolden) [inline examples/WarpSquare.py:5]: Import statement

**Review by Darylgolden — CHANGES_REQUESTED** [review]

File: examples/ContinuousMotion.py (hunk 1 above)
Comment (naveen521kk) [reply examples/ContinuousMotion.py:5]: It's added in the rendered page it there is none here.

Comment (Darylgolden) [reply examples/ContinuousMotion.py:5]: Then the other examples should not have import statements.

Comment (naveen521kk) [reply examples/ContinuousMotion.py:5]: hmm, will add them then

Comment (Darylgolden) [reply examples/ContinuousMotion.py:5]: In line with the docs, there should also only be one blank line between the import statement and the class

Comment (Darylgolden) [reply examples/ContinuousMotion.py:5]: > In line with the docs, there should also only be one blank line between the import statement and the class

^

Comment (naveen521kk) [reply examples/ContinuousMotion.py:5]: It's actually black which formatted the file like that.

Comment (Darylgolden) [reply examples/ContinuousMotion.py:5]: Is black insistent about the two lines of whitespace?

Comment (naveen521kk) [reply examples/ContinuousMotion.py:5]: Yeah

**Review by Darylgolden — APPROVED** [review]
LGTM

**Comment by naveen521kk** [comment]
Looks like it's better to remove import statements from all the examples in files? It looks odd when deployed.

**Pull request merged**